src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw

```diff
@@ -3183,4 +3183,7 @@ Please specify one of them using the --source option to proceed.</value>
   <data name="FontVersion" xml:space="preserve">
     <value>Version</value>
   </data>
-</root>
+  <data name="ConfigurationModules" xml:space="preserve">
+    <value>Configuration Modules</value>

```

Comment:
Nit: I'm not sure how clear this will be for the translators. I would add a `<comment>` like "PowerShell Modules that are used for the Configuration feature"